tests/utils/test__select_column.py

```diff
@@ -170,96 +191,88 @@ def test_strings(df1):
 
 def test_slice(df1):
     """Test _select_columns function on slices."""
-    assert_index_equal(
-        _select_columns(slice("code", "code2"), df1),
-        df1.loc[:, slice("code", "code2")].columns,
-    )
-    assert_index_equal(
-        _select_columns(slice("code2", None), df1),
-        df1.loc[:, slice("code2", None)].columns,
-    )
-    assert_index_equal(
-        _select_columns(slice(None, "code2"), df1),
-        df1.loc[:, slice(None, "code2")].columns,
-    )
-    assert_index_equal(_select_columns(slice(None, None), df1), df1.columns)
-    assert_index_equal(
-        _select_columns(slice(None, None, 2), df1),
-        df1.loc[:, slice(None, None, 2)].columns,
-    )
+    assert _select_columns(slice("code", "code2"), df1), df1.loc[
+        :, slice("code", "code2")
+    ].columns.tolist()
```

Comment:
@samukweku, I think I'm having a bit of trouble following this test, can you help me a bit here? Are you trying to assert equality between the result of `_select_columns` (in the first item position) and the second item? If so, I think you'll need to use the `assert pos1 == pos2` pattern, unless I'm missing something with the assert statement.

Reply:
realised assert does not need the `==` (it worked :) ), but I have fixed it in the accepted format.